src/neuroglancer/whats_new/whats_new.ts

```diff
@@ -24,15 +24,22 @@ const generateWhatsNew = (GHCommits: string[] = []) => {
 };
 
 export const findWhatsNew = async (viewer: Viewer) => {
-  let url =
-          `https://script.google.com/macros/s/AKfycbzVt6TLlJonmfU0EKTZVthi9pbM9dY1TYfTIH985tLUc8TZ5BNG/exec`,
-      WNCommits = JSON.parse(localStorage.getItem('WNCommits') || '[]'), headers = {
-        'Content-Type': 'text/plain;charset=utf-8',
-      },
-      body = JSON.stringify({
-        path: updateFName
-        // since: (WNCommits.length) ? WNCommits[0].commit.author.date : void(0)
-      });
+  const url =
+      `https://script.google.com/macros/s/AKfycbzVt6TLlJonmfU0EKTZVthi9pbM9dY1TYfTIH985tLUc8TZ5BNG/exec`;
+  const newUser = !localStorage.getItem('ng-newuser');
+  if (newUser) {
+    localStorage.setItem('ng-newuser', '1');
+    let description = (require('../../../NEW_USER.md')) || '';
+    return new WhatsNewDialog(viewer, description, {center: true});
+  }
+  const WNCommits = JSON.parse(localStorage.getItem('WNCommits') || '[]');
+  const headers = {
+    'Content-Type': 'text/plain;charset=utf-8',
+  };
+  const body = JSON.stringify({
+    path: updateFName
+    // since: (WNCommits.length) ? WNCommits[0].commit.author.date : void(0)
```

Comment:
Is this still a useful comment to keep hanging around?

Reply:
The functionality is going to be moved to ng-extend, whereas these requested changes will be moved to #431